lib/whitehall_importer/create_revision.rb

```diff
@@ -119,7 +119,12 @@ def tags(associations)
     def find_or_create_image_revisions(images)
       images.reduce([]) do |memo, image|
         already_imported = WhitehallMigration::AssetImport.find_by(original_asset_url: image["url"])
-        revision = if already_imported
+        revision = if already_imported &&
+            (already_imported.image_revision.alt_text != image["alt_text"] ||
+             already_imported.image_revision.caption != image["caption"])
+                     WhitehallImporter::CreateImageRevision
+                      .call(document_import, image, memo.map(&:filename), already_imported.image_revision.image)
+                   elsif already_imported
                      already_imported.image_revision
                    else
                      WhitehallImporter::CreateImageRevision
```

Comment:
I wonder if we should move all of this into a method since it's quite a lot of logic for a loop - or maybe the alt check may feel like the need for a new method.

Reply:
Moved all this logic into a new method and just calling the method within the loop.  Looks much cleaner now.